### PR TITLE
Addition of new class Space:Wing and ODR.

### PR DESCRIPTION
**Pull request by PeteHart**
Suggestion to new class Space:Wing and associated ODR. Comments appreciated.

The new class is introduced as brick:Location is being deprecated in 1.4.

Note introduction of new folder for rec:SubBuilding class to accomodate new rec:Wing subclass.

Also note the recommendation to use rec:SubBuilding for new applications.

### CODE REVIEW

File: OntologyDecisionRecords/ODR 004 Introduction of Space:Wing.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# ODR 004 Introduction of Space:Wing
+
+## Context
+
+With the release of Brick 1.4 the brick:Location domain will be deprecated in favour of rec:Space. All specific Brick location classes but brick:Wing are already covered in REC. This ODR introduces the building Wing concept in REC for future compatibility reasons.
```
Comment (cbupp) [inline OntologyDecisionRecords/ODR 004 Introduction of Space:Wing.md]: The wording is hard to follow. Try:

With the release of Brick 1.4 the `brick:Location` domain will be deprecated in favour of `rec:Space`. All `Brick:location` classes are already covered in REC except for `brick:Wing`. This ODR introduces the Wing concept in REC for future compatibility reasons.

File: OntologyDecisionRecords/ODR 004 Introduction of Space:Wing.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+## Decision
+
+An new class rec:Wing is introduced as subclass to the rec:SubBuilding.
```
Comment (cbupp) [inline OntologyDecisionRecords/ODR 004 Introduction of Space:Wing.md]: "A" not "An"

**Comment by PeteHart** [comment]
Have updated the ODR but have in parallel received feedback that several deprecated brick:Locations are missing clear replacements in REC. Discuss further in Technical Committe.
